typings/internal-ambient.d.ts

```diff
@@ -210,6 +210,17 @@ declare namespace NodeJS {
     _linkedBinding(name: 'electron_browser_desktop_capturer'): {
       createDesktopCapturer(): ElectronInternal.DesktopCapturer;
     };
+    _linkedBinding(name: 'electron_browser_dialog'): {
+      /** only available on mac and windows */
+      showCertificateTrustDialog?: (window: BrowserWindow, settings: CertificateTrustDialogOptions, message: string) => void;
+      showErrorBox: (...args: any[]) => void;
```

Comment:
The return type seems to be wrong, build is failing:

```
ERROR in /Users/distiller/project/src/electron/lib/browser/api/dialog.ts
./lib/browser/api/dialog.ts
[tsl] ERROR in /Users/distiller/project/src/electron/lib/browser/api/dialog.ts(228,3)
      TS2322: Type 'void' is not assignable to type 'OpenDialogReturnValue'.
```

Reply:
ok, thank you, I attempted to fix in 91e3ffa38b354b0252635ac2d18adce9eb58e57a

If that doesn't work, maybe I'll pull these changes out.  it was a nice-to-have to update the internal types but it's getting in the way of the core fix in this PR.